apps/e2e/cypress/e2e/instruments.cy.ts

```diff
@@ -356,7 +357,8 @@ context('Instrument tests', () => {
         userId: scientist2.id,
       });
 
-      cy.login(scientist2);
+      cy.login('user2');

```

Comment:
A lot of the modifed `cy.login` calls add a second parameter, but there're a couple in this file that don't. I assume it works without, but should it have a second parameter for consistency?

Reply:
The second parameter as when you login it logs you in as the highest permissions level and since I made `user1` an instrument sci now we need to tell cypress to log in as a user which is why I added the second parameter. For consistency should we add the second parameter throughout all the tests?

Reply:
I'll open a issue for this maybe? Would like to get this PR in.